<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompressWindow</class>
 <widget class="QDialog" name="CompressWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frog Archiver - Compress Files</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>File List</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item>
    <widget class="QPushButton" name="addFilesButton">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(252, 233, 79);</string>
     </property>
     <property name="text">
      <string>Add Files</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="clearListButton">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(164, 0, 0);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Clear Files</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Save Location</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="saveLocation">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="saveAsButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="embedButton">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(78, 154, 6);
qproperty-icon:: url(:/assets/1455813976_add-files-to-archive.png)</string>
     </property>
     <property name="text">
      <string>Compress</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit">
     <property name="styleSheet">
      <string notr="true">color: rgb(78, 154, 6);</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
